src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,10 @@
+.forecast-details {
+  display: flex;
+  flex-direction: column;  
+  flex: 1 1 auto;
+}
+
+.forecast-details > h2 {
```

Comment:
Instead of making as specific selector like this why now just add a classname to the h2?